lib/daru/vector.rb

```diff
@@ -830,6 +830,30 @@ def to_matrix axis=:horizontal
       end
     end
 
+    # Convert vector to nmatrix object
+    # @param [Symbol] axis :horizontal or :vertical
+    # @return [NMatrix] NMatrix object containing all values of the vector
+    # @example
+    #   dv = Daru::Vector.new [1, 2, 3]
+    #   dv.to_nmatrix
+    #   # =>
+    #   # [
+    #   #   [1, 2, 3] ]
+    def to_nmatrix axis=:horizontal
+      raise ArgumentError, 'Can not convert to nmatrix'\
+        'because the vector is numeric' unless numeric? && !include?(nil)
+
```

Comment:
No let it be. `nil` represents missing data, which can be numeric. Moreover, stats method work when missing data is present and they will malfunction if `numeric?` does not work the way it does.